File: backend/src/main/java/com/meetup/meetup/entity/Event.java
package com.meetup.meetup.entity;


import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.util.Calendar;
import java.util.List;

@Data
public class Event {

    private int eventId;
    private String name;
    private String eventDate;
    private String description;
    private int periodicityId;
    private EventPeriodicity periodicity;
    private String place;
    private int eventTypeId;
    private EventType eventType;
    @JsonProperty
    private boolean isDraft;
    private int folderId;
    private String imageFilepath;
    private int ownerId;

    private List<User> participants;

}

File: frontend/src/main/frontend/src/app/events/event.ts
import {Profile} from "../account/profile";
export class Evento{
  eventId : number;
  name : string;
  eventDate : string;
  description : string;
  periodicityId : number;
  periodicity : string;
  place : string;
  eventTypeId : number;
  eventType : string;
  isDraft : boolean;
  folderId : number;
  imageFilepath : string;
  ownerId : number;
  participants : Profile[]
}
